src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,3 @@
+package core.basesyntax;
+
+enum Colors { YELLOW, PINK, WHITE, ORANGE, BLUE, GREEN, BROWN, GREY, BLACK, RED }
```

Comment:
naming convention: https://mate-academy.github.io/style-guides/java/java.html#s4.8.1-enum-classes
```suggestion
enum Color { YELLOW, PINK, WHITE, ORANGE, BLUE, GREEN, BROWN, GREY, BLACK, RED }
```